### PR TITLE
[hotfix][streaming] Report correct value of currentProcessingTime in the exception

### PR DESCRIPTION
**Pull request by pnowojski**
This is a trivial logging change without test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

File: flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperator.java [hunk 1]
```diff
@@ -319,11 +319,14 @@ public void merge(W mergeResult,
 									"event-time window cannot become earlier than the current watermark " +
 									"by merging. Current watermark: " + internalTimerService.currentWatermark() +
 									" window: " + mergeResult);
-						} else if (!windowAssigner.isEventTime() && mergeResult.maxTimestamp() <= internalTimerService.currentProcessingTime()) {
-							throw new UnsupportedOperationException("The end timestamp of a " +
+						} else {
```
Comment (aljoscha) [inline flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperator.java]: This introduces an additional call to `internalTimerService.currentProcessingTime()`, which does a syscall, if I'm not mistaken, for the current time every time the else branch is taken.

We should probably do the `!windowAssigner.isEventTime()` check in the else and then retrieve the current processing time within the else block.

**Comment by pnowojski** [comment]
Yes, good catch. Thanks for spotting this.

**Comment by pnowojski** [comment]
Thanks for the quick review @aljoscha. Will merge it with green travis.

**Comment by pnowojski** [comment]
Merged

**Pull request merged**